CDTDatastore/touchdb/TD_Database+Insertion.m

```diff
@@ -369,7 +374,7 @@ - (TD_Revision*)putRevision:(TD_Revision*)rev
             // Inserting first revision, with docID given (PUT):
             if (docNumericID <= 0) {
                 // Doc ID doesn't exist at all; create it:
-                docNumericID = [self insertDocumentID:docID inDatabase:db];
+                docNumericID = [self insertDocumentID:docID inDatabase:db error:NULL];
```

Comment:
Wouldn't we be better making use of the error to differentiate between db full errors and general db errors as done elsewhere?